### ISSUE TITLE
Removing lint filter options causes the editor to crash

### ISSUE THREAD

**Issue by NilsJacobsen** [inlang/inlang#542]
## Problem
If you remove options from lint filter -> editor crashes
https://www.loom.com/share/e44d00f153da48a480555f813ecea6bf

## Steps to reproduce
Open editor: https://inlang.com/editor/github.com/jannesblobel/osmosis-frontend

### Expected behavior
It should just apply it, similar to the language filter

**Config:**
```typescript
/**
 * The inlang config is the entry point for every inlang app.
 *
 * More information about the config is available here
 * https://inlang.com/documentation/config
 */
export async function defineConfig(env) {
    // importing inlang-plugin-json from https://github.com/samuelstroschein/inlang-plugin-json
    const plugin = await env.$import(
        "https://cdn.jsdelivr.net/gh/samuelstroschein/inlang-plugin-json@1/dist/index.js"
    );
    const { standardLintRules } = await env.$import(
        "https://cdn.jsdelivr.net/gh/inlang/standard-lint-rules@1/dist/index.js"
      );
    const pluginConfig = {
        pathPattern: "./packages/web/localizations/{language}.json",
    };

    return {
        referenceLanguage: "en",
        languages: await plugin.getLanguages({
          ...env,
          pluginConfig,
        }),
        readResources: (args) => {
          return plugin.readResources({ ...args, ...env, pluginConfig });
        },
        writeResources: (args) => {
          return plugin.writeResources({ ...args, ...env, pluginConfig });
        },
        lint: {
          rules: [standardLintRules()],
        },
    };
}
```

It works in the example file.

**Comment by NilsJacobsen** [comment]
@ivanhofer do you see a problem with the config?

**Comment by NilsJacobsen** [comment]
I'm gonna check if there is a problem in the frontend.

**Comment by NilsJacobsen** [comment]
**Update:**
This issue is most likely a rendering issue. When removing all filters each Message and Pattern is gonna be rerendered. When we only set it to one or two languages then it works after a sec. We should work on the lazy loading to fix this.

**Comment by samuelstroschein** [comment]
I'll take this one. The bug is definitely caused by re-rendering as @NilsJacobsen already found out.

Shoelace/Web Components don't play nicely with Solids reactive system. On every change, Solid re-renders web components. We would likely not have this problem if we would use Solid native components.  

The shoelace textarea of patterns calls a height function which takes a whopping 40 seconds on an M1 Mac to load. 

<img width="1385" alt="CleanShot 2023-04-13 at 15 12 12@2x" src="https://user-images.githubusercontent.com/35429197/231769882-b6aa404d-14d8-4bc8-8638-66603a461201.png">

**Comment by NilsJacobsen** [comment]
https://user-images.githubusercontent.com/58360188/231814335-be82a7a9-be7c-43cf-9353-cd645b5608a0.mov

Still the same problem

**Comment by samuelstroschein** [comment]
Weird. Sometimes it works, sometimes it doesn't. Will look into this tomorrow!


https://user-images.githubusercontent.com/35429197/231865490-1d192ea4-7df4-41a6-9aef-897bf178c1d1.mp4

**Comment by samuelstroschein** [comment]
closed by #555
